File: SwimMasterCS/SwimMaster/SwimMaster/domain/Competition.cs
namespace SwimMaster.domain
{
    public class Competition : Entity<int>
    {
        private int distance;
        private string style;
        public Competition(int distance, string style) {
            this.distance = distance;
            this.style = style;
        }

        public int Distance { get => distance; set => distance = value; }
        public string Style { get => style; set => style = value; }

        public override string ToString()
        {
            return style + " " + distance.ToString()+"m";
        }
    }
}

File: SwimMasterJavaNW/SwimMasterModel/src/main/java/sm/model/Competition.java
package sm.model;

public class Competition extends Entity<Integer>{
    private int distance;
    private CompetitionStyle style;

    public Competition(int distance, CompetitionStyle style) {
        this.distance = distance;
        this.style = style;
    }

    public int getDistance() {
        return distance;
    }

    public void setDistance(int distance) {
        this.distance = distance;
    }

    public CompetitionStyle getStyle() {
        return style;
    }

    public void setStyle(CompetitionStyle style) {
        this.style = style;
    }

    @Override
    public String toString() {
        return this.style + " - " + this.distance +"m";
    }
}
